web/src/app/tree/tree-draw/tree-draw.component.html

```diff
@@ -0,0 +1,3 @@
+<div id="sample">
+  <div id="myDiagramDiv" style="background-color: #F8F8F8; border: solid 1px black; width:100%; height:600px;"></div>
```

Comment:
It seems strange sizing, but idk if it works that's ok. Why height is not in %

Reply:
It works pretty good on different devices with different sizes, it doesn't work at all with percentage value. I found it somewhere like that not the first time.